sources/config.yaml

```diff
@@ -0,0 +1,69 @@
+sources:
+  - Epilogue.glyphs
+  - Epilogue-Italic.glyphs
+axisOrder:
+  - wght
+  - ital
+familyName: "Epilogue"
+stat:
+    Epilogue[wght].ttf:
+    - name: Weight
+      tag: wght
+      values:
+      - name: Thin
+        value: 100
+      - name: ExtraLight
+        value: 200
+      - name: Light
+        value: 300
+      - name: Regular
+        value: 400
+        linkedValue: 700
+        flags: 2
+      - name: Medium
+        value: 500
+      - name: SemiBold
+        value: 600
+      - name: Bold
+        value: 700
+      - name: ExtraBold
+        value: 800
+      - name: Black
+        value: 900
+    - name: Italic
+      tag: ital
+      values:
+      - name: Roman
+        value: 0
+        linkedValue: 1
+        flags: 2
+    Epilogue-Italic[wght].ttf:
+    - name: Weight
+      tag: wght
+      values:
+      - name: Thin
+        value: 100
+      - name: ExtraLight
+        value: 200
+      - name: Light
+        value: 300
+      - name: Regular
+        value: 400
+        linkedValue: 700
+        flags: 2
+      - name: Medium
+        value: 500
+      - name: SemiBold
+        value: 600
+      - name: Bold
+        value: 700
+      - name: ExtraBold
+        value: 800
+      - name: Black
+        value: 900
+    - name: Italic
+      tag: ital
+      values:
+      - name: Italic
+        value: 1
+autohintTTF: False
```

Comment:
@aaronbell we actually want the static ttf to be hinted. VF won't be auto-hinted even if this parameter is True (I think)

Reply:
Oh is that right? I had understood it to mean autohinting all ttf files (including variable). :/

Reply:
I also have doubt so better check with the gftools guys, but from what I see in the other fonts I have build, only the statics gets autohinted

Reply:
Ugg. Well that means a lot of fixing for me!

Reply:
Simon just confirmed we really can't hint VF with TTFautohint…

Reply:
Just updated. Should be resolved now.